backend/src/crunchbase.js

```diff
@@ -0,0 +1,85 @@
+const axios = require("axios")
+
+const { getUrlDomain, camelizeKeys } = require("./utils")
+
+const API_KEY = process.env.CRUNCHBASE_API_KEY
+const ODM_ORGS_URL = "https://api.crunchbase.com/v3.1/odm-organizations"
+
+// Accepts an organization and returns an enriched version of that organization
+// as well as a raw Crunchbase enrichment object.
+// TODO: We will want to fall back to search by Name at some point
+// TODO: We will want to handle multiple results at some point
+async function crunchbaseEnrich({ name, homepage, crunchbase, twitter, linkedin }) {
+  const domain = getUrlDomain(homepage)
+  const results = await fetchCrunchbase({ domain })
+
+  if (results.length == 0) {
+    console.log(`crunchbaseEnrich: no results for Organization '${name}' domain: ${domain}`)
+  } else if (results.length > 1) {
+    console.log(`crunchbaseEnrich: multiple results for Organization '${name}' domain: ${domain}`, results)
+  } else {
+    const result = results[0]
+
+    // TODO: We can implement a more robust confidence check here, using other
+    // attributes like social media profiles to determine if we have the correct
+    // result. Name frequently doesn't match due to minor inconsistencies in the
+    // spelling or formatting
+    if (result.name === name) {
+      return result
+    } else {
+      console.log(`crunchbaseEnrich: name mismatch for Organization '${name}' got ${result.name}`)
+    }
+  }
+}
+
+// Accepts an OrganizationSummary from Crunchbase and returns a populated
+// Airtable Organization object
+function mapCrunchbase({
+  webPath, primaryRole, shortDescription, profileImageUrl,
+  facebookUrl, twitterUrl, linkedinUrl, cityName, regionName, countryCode,
+}) {
+  let role
+
+  if (primaryRole === "investor") role = "Capital"
+
+  return {
+    "Tagline Override": shortDescription,
+    Logo: profileImageUrl && [{ url: profileImageUrl }],
+    "Crunchbase URL Override": `https://crunchbase.com/${webPath}`,
+    Role: role,
+    Facebook: facebookUrl,
+    "Twitter Override": twitterUrl,
+    "LinkedIn Override": linkedinUrl,
+    "HQ Location Override": [cityName, regionName, countryCode].filter(x => x).join(", "),
```

Comment:
description would be nice as well.

Reply:
@jasonLaster we only get the short description from Crunchbase, mapped to `Tagline`